common/install_health_checks.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
+
+
 set -e
 
-# Set the azhc version
-AZHC_VERSION=$(jq -r '.azhc."'"$DISTRIBUTION"'".version' <<< $COMPONENT_VERSIONS)
+AZHC_VERSION=v0.4.2
```

Comment:
We should be using the version in requirements.json. No more hardcoding versions in files

Reply:
sounds good let me modify

Reply:
addressed